rasa/utils/tensorflow/temp_keras_modules.py

```diff
@@ -0,0 +1,265 @@
+import copy
+from typing import (
+    List,
+    Dict,
+    Union,
+    Optional,
+    Any,
+)
+
+import numpy as np
+import tensorflow as tf
+
+from tensorflow.python.keras.callbacks import Callback
+from tensorflow.python.keras.engine import training
+from tensorflow.python.keras.engine import training_utils
+from tensorflow.python.keras.engine import data_adapter
+from tensorflow.python.keras.utils import version_utils
+from tensorflow.python.eager import context
+from tensorflow.python.keras.engine.data_adapter import DataHandler
+
+
+# noinspection PyMethodOverriding
+class TmpKerasModel(tf.keras.models.Model):
+    """Temporary solution. Keras model that uses a custom data adapter inside fit."""
```

Comment:
Variable names like `x` and `y` in this class are weird and say nothing at all, it's constructor is too long, etc etc but I guess it doesn't matter much since this code is temporary and will be removed later anyway?

Reply:
this is a copy paste from keras, we're not changing variables in there. Besides `x` and `y` are perfect names here

Reply:
Yeah, would not modify this file as it is not "our" code. As mentioned in the docstring this is just a temporary solution as long as our PR on tensorflow side is not merged. The code is directly taken from tensorflow, just two lines are adapted.

Reply:
Ok, no problem :)